packages/graphiql/src/components/ToolbarMenu.tsx

```diff
@@ -75,7 +75,7 @@ export class ToolbarMenu extends React.Component<
     }
   }
 
-  handleClick(e: MouseEvent<HTMLAnchorElement>) {
+  handleClick(e: MouseEvent | React.MouseEvent<HTMLAnchorElement>) {
```

Comment:
ah yes, i remember setting these more specifically when we tried this PR nearly a year ago
I don't think theres a need for a union here, it should always be a React.MouseEvent<> correct?

Reply:
These are also being used as native listeners here: https://github.com/rohit-gohri/graphiql/blob/09f4d0b6e6b1c8ccc850d39cfae62c1fd14974b2/packages/graphiql/src/components/ToolbarMenu.tsx#L67

Reply:
ohh, right, good observation